src/OrchardCore.Modules/OrchardCore.Users/TimeZone/Services/UserTimeZoneService.cs

```diff
@@ -1,38 +1,36 @@
-using Microsoft.AspNetCore.Http;
-using Microsoft.AspNetCore.Identity;
 using Microsoft.Extensions.Caching.Distributed;
-using Microsoft.Extensions.DependencyInjection;
 using OrchardCore.Entities;
 using OrchardCore.Modules;
 using OrchardCore.Users.Models;
 using OrchardCore.Users.TimeZone.Models;
 
 namespace OrchardCore.Users.TimeZone.Services;
 
-public class UserTimeZoneService
+/// <summary>
+/// Represents a time zone service for a user.
+/// </summary>
+public class UserTimeZoneService : IUserTimeZoneService
 {
-    private const string EmptyTimeZone = "empty";
     private const string CacheKey = "UserTimeZone/";
+    private const string EmptyTimeZone = "empty";
+
     private static readonly DistributedCacheEntryOptions _slidingExpiration = new() { SlidingExpiration = TimeSpan.FromHours(1) };
 
     private readonly IClock _clock;
     private readonly IDistributedCache _distributedCache;
-    private readonly IHttpContextAccessor _httpContextAccessor;
 
-    public UserTimeZoneService(
-        IClock clock,
-        IDistributedCache distributedCache,
-        IHttpContextAccessor httpContextAccessor
-    )
+    public UserTimeZoneService(IClock clock, IDistributedCache distributedCache)
```

Comment:
Span the service across multiple line

Reply:
Usually, we did that for more than two params

Reply:
We do for more than 1. More than one will push the code too far to the right. Vertical coding is much more equable specially when we do PR reviews